tests/test_sensor.py

```
@@ -13,116 +80,100 @@ class TestUpdateRecurringAlarm:
     to integers, which would always be True, causing incorrect alarm scheduling.
     """
 
+    def _create_sensor_with_method(self):
+        """Create a minimal object with the _update_recurring_alarm method.
+
+        Instead of importing the full class (which has complex dependencies),
+        we recreate just the method logic for testing.
+        """
+
+        class MinimalSensor:
```

Pylint (4.0.4)

[refactor] Too few public methods (0/2)

(R0903)